ios-target-app/Home/Models/Target.swift

```diff
@@ -0,0 +1,22 @@
+//
+//  Target.swift
+//  ios-target-app
+//
+//  Created by Raul Piñeres Carrera on 1/11/22.
+//
+
+import Foundation
+
+struct Target: Codable {
+  let id: Int
+  let title: String
+  let latitude, longitude, radius: Double
+  let topicId: Int
+  
+  private enum CodingKeys: String, CodingKey {
+    case id
+    case title
+    case latitude, longitude, radius
```

Comment:
pls keep consistency, either you split them all in new lines or join the keys that don't have custom rawValues